api/api/controllers/default_controller.py

```diff
@@ -0,0 +1,39 @@
+# # Copyright (C) 2015-2019, Wazuh Inc.
+# # Created by Wazuh, Inc. <info@wazuh.com>.
+# # This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import logging
+import os
+import socket
+import time
+
+import yaml
+
+from api import __path__ as api_path
+from api.models.basic_info import BasicInfo
+from api.util import exception_handler
+
+logger = logging.getLogger('wazuh')
+
+
+@exception_handler
+def default_info():
+    """Get basicinfo
+
+    Returns various basic information about the API
+    """
+    with open(os.path.join(api_path[0], 'spec', 'spec.yaml'), 'r') as stream:
+        info_data = yaml.load(stream)
```

Comment:
Is there any way to take the spec directly loaded from connexion instead of reloading the file?

Reply:
Ok, we didn't manage to find a better way, so let's go on with this approach.